indico/modules/rb/tasks.py

```diff
@@ -111,9 +62,17 @@ def roombooking_occurrences():
 
     try:
         for occ in occurrences:
-            notify_upcoming_occurrence(occ)
-            occ.notification_sent = True
-            if occ.reservation.repeat_frequency == RepeatFrequency.DAY:
-                occ.reservation.occurrences.update({'notification_sent': True})
+            same_user_occurrences = []
+            for other in occurrences:
```

Comment:
No I haven't, since its celery task running every 8 hours I thought the performance is not that important. But I guess you are right ...